### ISSUE TITLE
Ctrl + C quits with exception

### ISSUE THREAD

**Issue by diimdeep** [facebook/PathPicker#42]
```
Traceback (most recent call last):
  File "/usr/local/Cellar/fpp/0.5.3/libexec/src/choose.py", line 58, in <module>
    curses.wrapper(doProgram)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/curses/wrapper.py", line 43, in wrapper
    return func(stdscr, *args, **kwds)
  File "/usr/local/Cellar/fpp/0.5.3/libexec/src/choose.py", line 30, in doProgram
    screen.control()
  File "/usr/local/Cellar/fpp/0.5.3/libexec/src/screenControl.py", line 262, in control
    inKey = self.getKey()
  File "/usr/local/Cellar/fpp/0.5.3/libexec/src/screenControl.py", line 490, in getKey
    charCode = self.stdscr.getch()
KeyboardInterrupt
```

**Comment by davidcox143** [comment]
This was actually fixed by @pcottle in commit [79b3d25](https://github.com/facebook/PathPicker/commit/79b3d2544749b810e87d42108d58429b0c10d7ca). 

Issue should be marked as closed.

**Comment by pcottle** [comment]
Thanks @davidcox143 ! Yep this must have been a duplicate
